Formula/libgsm.rb

```diff
@@ -35,7 +41,13 @@ def install
     man3.mkpath
 
     # Dynamic library must be built first
-    system "make", "lib/libgsm.1.0.13.dylib",
+    inreplace "Makefile", "@@SOVERSION@@", version
+    library = "libgsm.#{version}.dylib"
+    on_linux do
+      library = "libgsm.so"
```

Comment:
Why version it on one OS and not the other? Also, a helper that spits out `dylib` or `so` given an optional `version` would be good here, I think.

Reply:
`shared_library` already accepts an optional `version` parameter. However it looks like the OS-specific code is needed here since the .dylib is versioned but the .so is not.

Reply:
Why not submit a patch upstream to rectify that? That doesn't seem like something _anyone_ would be happy with.